src/AwsSnsSmsProvider.php

```diff
@@ -15,7 +15,7 @@ class AwsSnsSmsProvider extends SMSProvider
     public function getClient()
     {
         if (!isset($this->client)) {
-            $awsSettings = AwsSettings::singleton();
+            $awsSettings = AwsSnsSettings::singleton();
```

Comment:
I think this is currently a breaking change unfortunately. but seems a shame to do a major version bump for such a small change. Can you think of anyway to allow region to be overriden but in a way that if Niacro was updated it would continue to work?